erpnext/regional/india/setup.py

```diff
@@ -19,6 +20,7 @@ def setup(company=None, patch=True):
 # TODO: for all countries
 def setup_company_independent_fixtures():
 	make_custom_fields()
+	make_property_setters()
```

Comment:
@ankush Property setter should not be created if called from the patch. Because, it will replace customer's customized series which might be within permitted length.
I have fixed this in develop and version-13 via https://github.com/frappe/erpnext/commit/aa809ceffef679fa8c12472f5dc51fb734c7438c. Can you please make that changes in v12 as well?